### PR TITLE
ranking: Finalize v1 backend (vacuum+observability)

### PR DESCRIPTION
**Pull request by Numbers88s**
This PR enables the last changes necessary to enable on dot-com/s2.

## Test plan
Tested manually.

rank

### CODE REVIEW

**Review by efritz — CHANGES_REQUESTED** [review]
Let me submit a PR onto this branch that will do vacuuming for the new tables.

**Comment by Numbers88s** [comment]
> 

For sure, thanks man.

File: monitoring/definitions/shared/codeintel_ranking.go [hunk 1]
```diff
@@ -66,6 +66,100 @@ func (codeIntelligence) NewRankingStoreGroup(containerName string) monitoring.Gr
...
+				Standard.Count("reference rows inserted")(ObservableConstructorOptions{
+					MetricNameRoot:        "codeintel_uploads_ranking_num_references_inserted",
+					MetricDescriptionRoot: "reference rows processed",
+				})(containerName, monitoring.ObservableOwnerCodeIntel).WithNoAlerts(`
+					The number of reference rows inserted into Postgres.
+				`).Observable(),
+			},
+
+			{
+				Standard.Count("reference rows processed")(ObservableConstructorOptions{
```
Comment (Strum355) [inline monitoring/definitions/shared/codeintel_ranking.go:101]: `Description: fmt.Sprintf("%s%s every %s", options.MetricDescriptionRoot, legend, options.RangeWindow.String()),`
Means this becomes `reference rows processed reference rows processed ...`, where `legend` is the input to `Standard.Count`

Comment (efritz) [reply monitoring/definitions/shared/codeintel_ranking.go]: image

Comment (Strum355) [reply monitoring/definitions/shared/codeintel_ranking.go]: Being super pedantic, as a non-knowledgeable person looking at this, i have no idea what they are :^)

Comment (efritz) [reply monitoring/definitions/shared/codeintel_ranking.go]: LOOM INCOMING

**Review by efritz — APPROVED** [review]

**Pull request merged**